packages/core/src/utils/sourceMaps.ts

```diff
@@ -0,0 +1,21 @@
+import type { ExceptionStackFrame } from '..';
+import { globalObject } from '../globalObject';
+
+export const cachedBundleIdStackFrameMap = new Map<string, ExceptionStackFrame[]>();
+
+export function getBundleId(appName: string) {
+  return (globalObject as any)?.[`__faroBundleId_${appName}`];
+}
+
+export function getBundleIdStackMap() {
+  return (globalObject as any)?.[`__faroBundleIds`] as Map<Error, string> | undefined;
+}
+
+export function getBundleIdFromError(error: Error) {
```

Comment:
Clarification Question: What is this used for? Reading through the code, I am assuming having different bundles for different parts/slices of the app?